tests/ClientTest.php

```diff
@@ -17,7 +17,8 @@
 use Facebook\WebDriver\JavaScriptExecutor;
 use Facebook\WebDriver\WebDriver;
 use Facebook\WebDriver\WebDriverExpectedCondition;
-use Symfony\Component\BrowserKit\Client as BrowserKitClient;
+use LogicException;
```

Comment:
We use `\LogicException` instead of this in Symfony.

Reply:
As in, you mean not importing the exception class? Ok, will change that.